xformers/components/attention/global_tokens.py

```diff
@@ -26,7 +26,9 @@
 
 @dataclass
 class GlobalAttentionConfig(AttentionConfig):
-    attention_query_mask: torch.Tensor  # Mark the queries which have global attention
+    # Mark the queries which have global attention
+    # should be torch.Tensor, but OmegaConf does not support this typing
+    attention_query_mask: Any
```

Comment:
hmm, this is not too nice then, could it be something on which we can sync with @pradeep90 ? He should know one thing or two about type checking tensors :D

Reply:
@jieru-hu Could you explain what you mean by `OmegaConf` does not support this typing? What is the behavior you get?

Reply:
@pradeep90  - thanks for taking a look. OmegaConf right now does not support any Pytorch typing. I've reverted this change